src/io/iter_prefetcher.h

```diff
@@ -139,10 +139,10 @@ class PrefetcherIter : public IIterator<DataBatch> {
   PrefetcherParam param_;
   /*! \brief backend thread */
   dmlc::ThreadedIter<DataBatch> iter;
-
- private:
   /*! \brief internal batch loader */
   std::unique_ptr<IIterator<TBlobBatch> > loader_;
+
+ private:
```

Comment:
???

Reply:
this fixes a build error with caffe plugin.  Haibin had moved this into private, but CaffeDataIter needs it, so I moved it back